doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -100,7 +100,7 @@ putlog <text>
 putcmdlog <text>
 ^^^^^^^^^^^^^^^^
 
-  Description: sends text to the bot's logfile, marked as 'command' (c)
+Description: logs <text> to the logfile and partyline if the 'cmds' flag (c) is active via the 'logfile' config file setting and the '.console' partyline setting, respectively.
```

Comment:
Are the prefixing spaces removed by accident or intentional? Same for `putxferlog`.